src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Telemetry/IVsTelemetryService.cs

```diff
@@ -7,14 +7,9 @@ namespace Microsoft.VisualStudio.Telemetry
     /// <summary>
     /// Internal as we don't want anyone who depends on us to post events to our eventId.
     /// </summary>
-    internal interface ITelemetryService
+    internal interface IVsTelemetryService
     {
         /// <summary>
-        /// Posts a given telemetry event path to the telemetry service session for the program.
-        /// </summary>
-        void PostEvent(string eventName);
-
-        /// <summary>
         /// Posts a given telemetry operation to the telemetry service. <seealso cref="TelemetrySessionExtensions.PostOperation(TelemetrySession, string, TelemetryResult, string, TelemetryEventCorrelation[])"/>
         /// </summary>
         /// <param name="eventName">Name of the event.</param>
```

Comment:
I see that you say this elsewhere:

> Making this split enables Non-VS components to report telemetry and VS components to do advanced telemetry logging.

I think there should be one way to log telemetry across the codebase. If we need any advanced logging, we should abstract those concepts in the ITelemetryService and VSTelemetryService has a vs implementation of that. Having one for each layer will just cause confusion.